### PR TITLE
Add option to disable linting if .jshintrc is not found 

### PR DESCRIPTION
**Pull request by fixe**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Why?

**Comment by fixe** [comment]
Multiple projects, multiple linters.

File: index.js [hunk 1]
```diff
@@ -282,6 +286,10 @@ const registerEvents = () => {
 };
 
 export const config = plugin.config = {
+	validateOnlyWhenConfigured: {
```
Comment (sindresorhus) [inline index.js]: Needs a clearer name.

Comment (fixe) [inline index.js]: @sindresorhus any suggestions?

Comment (arthurvr) [inline index.js]: Ugh, this is a difficult one.

Comment (fixe) [inline index.js]: Should I close this? :-)

Comment (arthurvr) [inline index.js]: No, let's just think about another name. I wonder if @sindresorhus can come up with something :p

Comment (sindresorhus) [inline index.js]: Can't really think of anything else, but I guess we can keep this and rather explain what it does using a `title` prop. See the other option for example.

Comment (stevemao) [inline index.js]: linter-jscs has the same config
https://github.com/AtomLinter/linter-jscs/blob/master/index.js#L143
they call it `onlyConfig`

Comment (sindresorhus) [inline index.js]: Alright. Let's just copy that: https://github.com/AtomLinter/linter-jscs/blob/6a3ceaade90d03b9952eb266da2e5b878b729e60/index.js#L24-L29

@fixe

**Comment by stevemao** [comment]
Hey guys, just letting you know atom-linter is going to remove it in all their linters for performance reason.
https://github.com/atom-community/linter/issues/744

**Comment by sindresorhus** [comment]
@stevemao I don't see how that's relevant to this issue. They're talking about something completely different.

**Comment by stevemao** [comment]
Well I originally posted https://github.com/atom-community/linter/issues/818 but unfortunately the conversation in https://github.com/atom-community/linter/issues/744 has changed the focus a bit.

**Comment by sindresorhus** [comment]
ping @fixe :)

**Comment by fixe** [comment]
@sindresorhus done, sorry for the delay.

**Comment by sindresorhus** [comment]
Thanks! Available in 1.4.0 ;)

**Pull request closed without merging**